### PR TITLE
Show status across --all builds. Fix selftest.

### LINKED ISSUE #81: Change models/selftest/relu to some other model
Analysis is set to ignore `relu` instances when it runs. One of the two selftest models is just a `relu`, and so nothing happens when we run that script. Replace the `relu` with something that will actually get picked up by Analysis.

### PR DESCRIPTION
**Pull request by jeremyfowers**
Closes #81 #82 

New behavior:

Command:
```
(mla) jfowers@jfowers:~/mlagility$ benchit benchmark -s models/selftest --all
```

Output (notice how the two models are from two different files):

```
Models discovered during profiling:

model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          LinearTestModel (<class 'linear.LinearTestModel'>)
        Location:       models/selftest/linear.py, line 21
        Parameters:     110 (<0.1 MB)
        Hash:           d5b1df11
        Status:         Model successfully built!

model (executed 1x)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          TwoLayerTestModel (<class 'twolayer.TwoLayerTestModel'>)
        Location:       models/selftest/twolayer.py, line 24
        Parameters:     85 (<0.1 MB)
        Hash:           676c7c6d
        Status:         Model successfully built!
```

### CODE REVIEW

**Review by danielholanda — COMMENTED** [review]
Code looks good. I also tested this with scripts that contain multiple models and it is also working nicely there.

The only issue now is that we are getting a list of models without knowing where those models came from (we know where the model was instantiated, but we don't necessarily know the parent script). 

Here is what happens when I add [toxic-bert](https://huggingface.co/unitary/toxic-bert) to selftest :

```
Models discovered during profiling:

model (executed 1x - 0.02s)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          BertForSequenceClassification (<class 'transformers.models.bert.modeling_bert.BertForSequenceClassification'>)
        Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
        Parameters:     109,486,854 (208.8 MB)
        Hash:           9a3cce5a

model (executed 1x - 0.03s)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          RobertaForSequenceClassification (<class 'transformers.models.roberta.modeling_roberta.RobertaForSequenceClassification'>)
        Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
        Parameters:     124,657,936 (237.8 MB)
        Hash:           2c2ae382

model (executed 1x - 0.04s)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          XLMRobertaForSequenceClassification (<class 'transformers.models.xlm_roberta.modeling_xlm_roberta.XLMRobertaForSequenceClassification'>)
        Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
        Parameters:     278,055,952 (530.3 MB)
        Hash:           ba09c1b7

model (executed 1x - 0.00s)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          TwoLayerTestModel (<class 'twolayer.TwoLayerTestModel'>)
        Location:       models/selftest/twolayer.py, line 24
        Parameters:     85 (<0.1 MB)
        Hash:           676c7c6d

model (executed 4x - 0.00s)
        Model Type:     Pytorch (torch.nn.Module)
        Class:          LinearTestModel (<class 'linear.LinearTestModel'>)
        Location:       models/selftest/linear.py, line 33
        Parameters:     110 (<0.1 MB)
        Hash:           d5b1df11
```

As you can see, it is hard to understand where the first 3 models come from. 

Here is one suggestion:

```
Models discovered during profiling:

toxic_bert.py
    model (executed 1x - 0.02s)
            Model Type:     Pytorch (torch.nn.Module)
            Class:          BertForSequenceClassification (<class 'transformers.models.bert.modeling_bert.BertForSequenceClassification'>)
            Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
            Parameters:     109,486,854 (208.8 MB)
            Hash:           9a3cce5a

    model (executed 1x - 0.03s)
            Model Type:     Pytorch (torch.nn.Module)
            Class:          RobertaForSequenceClassification (<class 'transformers.models.roberta.modeling_roberta.RobertaForSequenceClassification'>)
            Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
            Parameters:     124,657,936 (237.8 MB)
            Hash:           2c2ae382

    model (executed 1x - 0.04s)
            Model Type:     Pytorch (torch.nn.Module)
            Class:          XLMRobertaForSequenceClassification (<class 'transformers.models.xlm_roberta.modeling_xlm_roberta.XLMRobertaForSequenceClassification'>)
            Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
            Parameters:     278,055,952 (530.3 MB)
            Hash:           ba09c1b7

twolayer.py
    model (executed 1x - 0.00s)
            Model Type:     Pytorch (torch.nn.Module)
            Class:          TwoLayerTestModel (<class 'twolayer.TwoLayerTestModel'>)
            Location:       models/selftest/twolayer.py, line 24
            Parameters:     85 (<0.1 MB)
            Hash:           676c7c6d

linear.py
    model (executed 4x - 0.00s)
            Model Type:     Pytorch (torch.nn.Module)
            Class:          LinearTestModel (<class 'linear.LinearTestModel'>)
            Location:       models/selftest/linear.py, line 33
            Parameters:     110 (<0.1 MB)
            Hash:           d5b1df11
```

**Comment by jeremyfowers** [comment]
> Code looks good. I also tested this with scripts that contain multiple models and it is also working nicely there.
> 
> The only issue now is that we are getting a list of models without knowing where those models came from (we know where the model was instantiated, but we don't necessarily know the parent script).
> 
> Here is what happens when I add [toxic-bert](https://huggingface.co/unitary/toxic-bert) to selftest :
> 
> ```
> Models discovered during profiling:
> 
> model (executed 1x - 0.02s)
>         Model Type:     Pytorch (torch.nn.Module)
>         Class:          BertForSequenceClassification (<class 'transformers.models.bert.modeling_bert.BertForSequenceClassification'>)
>         Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
>         Parameters:     109,486,854 (208.8 MB)
>         Hash:           9a3cce5a
> 
> model (executed 1x - 0.03s)
>         Model Type:     Pytorch (torch.nn.Module)
>         Class:          RobertaForSequenceClassification (<class 'transformers.models.roberta.modeling_roberta.RobertaForSequenceClassification'>)
>         Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
>         Parameters:     124,657,936 (237.8 MB)
>         Hash:           2c2ae382
> 
> model (executed 1x - 0.04s)
>         Model Type:     Pytorch (torch.nn.Module)
>         Class:          XLMRobertaForSequenceClassification (<class 'transformers.models.xlm_roberta.modeling_xlm_roberta.XLMRobertaForSequenceClassification'>)
>         Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
>         Parameters:     278,055,952 (530.3 MB)
>         Hash:           ba09c1b7
> 
> model (executed 1x - 0.00s)
>         Model Type:     Pytorch (torch.nn.Module)
>         Class:          TwoLayerTestModel (<class 'twolayer.TwoLayerTestModel'>)
>         Location:       models/selftest/twolayer.py, line 24
>         Parameters:     85 (<0.1 MB)
>         Hash:           676c7c6d
> 
> model (executed 4x - 0.00s)
>         Model Type:     Pytorch (torch.nn.Module)
>         Class:          LinearTestModel (<class 'linear.LinearTestModel'>)
>         Location:       models/selftest/linear.py, line 33
>         Parameters:     110 (<0.1 MB)
>         Hash:           d5b1df11
> ```
> 
> As you can see, it is hard to understand where the first 3 models come from.
> 
> Here is one suggestion:
> 
> ```
> Models discovered during profiling:
> 
> toxic_bert.py
>     model (executed 1x - 0.02s)
>             Model Type:     Pytorch (torch.nn.Module)
>             Class:          BertForSequenceClassification (<class 'transformers.models.bert.modeling_bert.BertForSequenceClassification'>)
>             Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
>             Parameters:     109,486,854 (208.8 MB)
>             Hash:           9a3cce5a
> 
>     model (executed 1x - 0.03s)
>             Model Type:     Pytorch (torch.nn.Module)
>             Class:          RobertaForSequenceClassification (<class 'transformers.models.roberta.modeling_roberta.RobertaForSequenceClassification'>)
>             Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
>             Parameters:     124,657,936 (237.8 MB)
>             Hash:           2c2ae382
> 
>     model (executed 1x - 0.04s)
>             Model Type:     Pytorch (torch.nn.Module)
>             Class:          XLMRobertaForSequenceClassification (<class 'transformers.models.xlm_roberta.modeling_xlm_roberta.XLMRobertaForSequenceClassification'>)
>             Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/transformers/modeling_utils.py, line 2245
>             Parameters:     278,055,952 (530.3 MB)
>             Hash:           ba09c1b7
> 
> twolayer.py
>     model (executed 1x - 0.00s)
>             Model Type:     Pytorch (torch.nn.Module)
>             Class:          TwoLayerTestModel (<class 'twolayer.TwoLayerTestModel'>)
>             Location:       models/selftest/twolayer.py, line 24
>             Parameters:     85 (<0.1 MB)
>             Hash:           676c7c6d
> 
> linear.py
>     model (executed 4x - 0.00s)
>             Model Type:     Pytorch (torch.nn.Module)
>             Class:          LinearTestModel (<class 'linear.LinearTestModel'>)
>             Location:       models/selftest/linear.py, line 33
>             Parameters:     110 (<0.1 MB)
>             Hash:           d5b1df11
> ```

Done!

File: examples/cli/hello_world.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 import torch
 
-torch.manual_seed(0)
+torch.manual_seed(1)
```
Comment (danielholanda) [inline examples/cli/hello_world.py:13]: why?

**Review by danielholanda — APPROVED** [review]

File: examples/cli/hello_world.py (hunk 1 above)
Comment (jeremyfowers) [reply examples/cli/hello_world.py:13]: So it doesn’t have an identical hash with TwoModels

**Pull request merged**